natalie/template3a-tarot.js

```diff
@@ -0,0 +1,46 @@
+// This IIFE should create a subclass (constructor TarotCard)
+//  of a superclass (constructor Card)
+var TarotCard = (function(){
```

Comment:
Your tarot cards look good. An improvement you could make, however, is making sure that if a card is not valid, it does not return a rank, or name or suit. Currently cards beyond id 55 are not considered valid, but still return results of their methods are called. Good work, overall!